### PR TITLE
PCR in-app invite

### PR DESCRIPTION
**Pull request by juliatan**
# Description

- **!! WIP - this needs to tie into backend which is currently on hold**
- Also need to look into whether we can have the BottomSheet go all the way to the bottom of the screen.
- Ticket: https://www.notion.so/joinzoe/In-App-PCR-Test-Invites-c2fabaa7be744a8591d25fb8b36aeb28

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [X] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

Jun-14-2021 17-56-44

## Checklist

- [ ] I have updated mockServer
- [ ] I've added analytics as relevant
- [ ] I have run `npm test`
- [ ] I have run `npm run test:i18n`

## Out of scope and potential follow-up

n/a

### CODE REVIEW

File: src/features/assessment/PcrTestLearnMoreScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  };
+
+  const renderBulletLine = (text: string) => (
+    <View style={{ flexDirection: 'row', marginBottom: 8 }}>
+      <RegularText style={{ fontSize: 4 }}>{'\u2B24'}</RegularText>
+      <RegularText style={{ flex: 1, paddingLeft: 16 }}>{text}</RegularText>
+    </View>
+  );
+
+  const renderSubBulletLine = (text: string) => (
```
Comment (bionara) [inline src/features/assessment/PcrTestLearnMoreScreen.tsx:23]: May be worth us making this (and above function) into components

File: package.json [hunk 2]
```diff
@@ -102,6 +102,7 @@
     "react-native-calendar-picker": "6.1.5",
     "react-native-config": "1.4.1",
     "react-native-easy-markdown": "2.0.0",
+    "react-native-elements": "^3.4.1",
```
Comment (bionara) [inline package.json:105]: wahoo

File: assets/lang/en.json [hunk 3]
```diff
@@ -473,7 +473,7 @@
     "q4-info-1": "Please only consider symptoms that are not explained by another reason.",
```
Comment (bionara) [inline assets/lang/en.json:473]: Don't forget to have the same keys in the es and sv locales.

File: src/features/assessment/WhereAreYouScreen.tsx [hunk 4]
```diff
@@ -82,14 +114,70 @@ function WhereAreYouScreen({ navigation, route }: IProps) {
...
+          <BrandedButton
+            onPress={() => openWebLink('https://www.gov.uk/get-coronavirus-test')}
+            style={styles.bookTestButton}
+          >
+            Book your test
+          </BrandedButton>
+        </View>
+      </BottomSheet>
+
+      <Modal transparent animationType="slide" onRequestClose={closeLearnMore} visible={modalVisible}>
```
Comment (bionara) [inline src/features/assessment/WhereAreYouScreen.tsx:144]: this is way better than the custom one we have. I'll make a cleaning ticket to replace existing if all are happy. Ace!

**Review by bionara — COMMENTED** [review]
Looks great so far. Love the new <Modal> element vs our clunky route-based one!

File: src/features/assessment/PcrTestLearnMoreScreen.tsx (hunk 1 above)
Comment (juliatan) [reply src/features/assessment/PcrTestLearnMoreScreen.tsx:23]: Yip can do!

File: assets/lang/en.json (hunk 3 above)
Comment (juliatan) [reply assets/lang/en.json:473]: Realised that we only want this for UK, so removed the internationalisation in a subsequent commit.

File: src/features/assessment/WhereAreYouScreen.tsx (hunk 4 above)
Comment (juliatan) [reply src/features/assessment/WhereAreYouScreen.tsx:144]: Yea, agree. Was a lot simpler to implement!

**Comment by bionara** [comment]
Closing PR as not on product roadmap anymore.

**Pull request closed without merging**